code/game/objects/items/devices/flashlight.dm

```diff
@@ -147,6 +147,16 @@
 	w_class = 1
 	raillight_compatible = FALSE
 
+/obj/item/device/flashlight/combat //as bright as rail lights, don't see why you need them to be mounted on a gun to be brighter
+	name = "combat flashlight"
+	desc = "A hand-held flashlight, with increased light for military usage."
+	icon = 'icons/obj/lighting.dmi'
+	icon_state = "flashlight_combat"
+	w_class = 2
+	on = FALSE
+	brightness_on = 7
+	raillight_compatible = FALSE
```

Comment:
There is already one at `code/modules/cm_marines/equipment/gear.dm`.

Just cut and paste it here for easier indexing and then apply the changes.
And there is no need to input new var values if they are the same of the parent (`w_class`, `on` and `icon` in this case)